### ISSUE TITLE
Could not upload bugreport

### ISSUE THREAD

**Issue by yschandra** [google/battery-historian#121]
On Latest I/O 2017 release of Battery Historian, when tried to upload bugreport nothing happens typically "Submit" button should appear which does not happen. But if I start battery historian with "--optimized=false" flag every thing works fine.

Note that this not in Docker environment.

**Comment by zfk0531** [comment]
Yes, I have the same problem. I setup with the latest Battery Historian, but I can not upload bugreport. When I hit "Browse" and open the ".txt" file, nothing happens.  I have tried to use "--optimized=false" flag, but it does not work either.

btw. I also do not use Docker.

**Comment by saviii** [comment]
Having the same issue on my end, using the latest Battery historian. Any update on a resolution or a possible workaround?

**Comment by 27DH** [comment]
I am also having the same issue.......

**Comment by dengguanyang** [comment]
in China the page use these were unreachabled
ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/hot-sneaks/jquery-ui.css
ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js
ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js
https://www.google.com/jsapi
www.gstatic.com/external_hosted/jquery_form/jquery.form.min.js

一句话就是被墙了，翻墙就可用了 @27DH

**Comment by lxzh** [comment]
> in China the page use these were unreachabled
> ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/hot-sneaks/jquery-ui.css
> ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js
> ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js
> https://www.google.com/jsapi
> [www.gstatic.com/external_hosted/jquery_form/jquery.form.min.js](http://www.gstatic.com/external_hosted/jquery_form/jquery.form.min.js)
> 
> 一句话就是被墙了，翻墙就可用了 @27DH

非常感谢，浏览文件后半天没反应，还以为哪里不对劲，翻墙后就好了
